### PR TITLE
Make cryo start with auto eject enabled

### PR DESCRIPTION
**Pull request by theo-3**
## What Does This PR Do
Makes cryo start with all eject options enabled.

## Why It's Good For The Game
Prevents people from getting stuck in cryo because someone didn't enable eject healthy.

## Changelog
:cl:
tweak: cryo now starts with auto-eject healthy and dead enabled
/:cl:

### CODE REVIEW

**Review by SteelSlayer — APPROVED** [review]

**Comment by theo-3** [comment]
Sorry,I didn't mean to close it

**Comment by theo-3** [comment]
I accidentally deleted that branch

**Review by Kyep — APPROVED** [review]

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@
 	var/mob/living/carbon/occupant = null
 	var/obj/item/reagent_containers/glass/beaker = null
 	/// Holds two bitflags, AUTO_EJECT_DEAD and AUTO_EJECT_HEALTHY. Used to determine if the cryo cell will auto-eject dead and/or completely health patients.
```
Comment (SabreML) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:20]: While you're here, could you also fix the typo on this comment?
```suggestion
	/// Holds two bitflags, AUTO_EJECT_DEAD and AUTO_EJECT_HEALTHY. Used to determine if the cryo cell will auto-eject dead and/or completely healthy patients.
```

**Comment by theo-3** [comment]
Done

**Review by SabreML — APPROVED** [review]
Thanks!

**Review by dearmochi — APPROVED** [review]

**Review by variableundefined — APPROVED** [review]
LGTM

**Pull request merged**